Firestore/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Unreleased
 
+# v0.12.4
+- [fixed] DocumentReference.setData(_:mergeFields:) can now delete fields using
+  FieldValue.delete().
+- [fixed] Firestore will now transparently recover from auth token expiration.
```

Comment:
Perhaps something like `Firestore will now recover from auth token expiration when the system clock is wrong`

Reply:
Done.